tutorial.md

```diff
@@ -11,7 +11,7 @@ If you have npm 5.2+ installed (or node), you can run zem directly to create the
 Let's create it in a directory called `hello-layer`.
 
 ```sh
-npx zem create hello-layer
+npm zem create hello-layer
```

Comment:
This is a utility that execute npm package executables. See:
https://github.com/npm/npx